pkg/kubelet/kubelet_node_status.go

```diff
@@ -56,6 +56,9 @@ const (
 	// maxNamesPerImageInNodeStatus is max number of names per image stored in
 	// the node status.
 	maxNamesPerImageInNodeStatus = 5
+
+	// mb is used to easily convert an int to an mb
+	mb = 1024 * 1024
```

Comment:
There must be a helper for this somewhere, right? We do these sorts of calculations for memory limits, don't we?
Consider risk of overflow if you ever use ints for GB-size values with this, they can be 32 bits (unless you only ever divide by `mb`?).
It's a nit but differentiating between MB and Mb is also sometimes important.